contracts/cctp/fees/SynapseCCTPFees.sol

```diff
@@ -0,0 +1,265 @@
+    bytes private constant SYMBOL_PREFIX = "CCTP.";
+    /// @dev Length of the mandatory prefix used for CCTP token symbols
+    uint256 private constant SYMBOL_PREFIX_LENGTH = 5;
+
+    // ══════════════════════════════════════════════════ STORAGE ══════════════════════════════════════════════════════
+
+    /// @notice Maps bridge token address into bridge token symbol
+    mapping(address => string) public tokenToSymbol;
+    /// @notice Maps bridge token symbol into bridge token address
+    mapping(string => address) public symbolToToken;
+    /// @notice Maps bridge token address into CCTP fee structure
+    mapping(address => CCTPFee) public feeStructures;
+    /// @notice Maps fee collector address into accumulated fees for a token
+    /// (feeCollector => (token => amount))
+    /// @dev Fee collector address of address(0) indicates that fees are accumulated by the Protocol
+    mapping(address => mapping(address => uint256)) public accumulatedFees;
+    /// @notice Maps Relayer address into collector address for accumulated Relayer's fees
+    /// @dev Default value of address(0) indicates that a Relayer's fees are accumulated by the Protocol
+    mapping(address => address) public relayerFeeCollectors;
+    /// @notice Protocol fee: percentage of the relayer fee that is collected by the Protocol
+    /// @dev Protocol collects the full fee amount, if the Relayer hasn't set a fee collector
+    uint256 public protocolFee;
+    /// @dev A list of all supported bridge tokens
+    EnumerableSet.AddressSet internal _bridgeTokens;
+
+    // ════════════════════════════════════════════════ ONLY OWNER ═════════════════════════════════════════════════════
+
+    /// @notice Adds a new token to the list of supported tokens, with the given symbol and fee structure.
+    /// @dev The symbol must start with "CCTP."
+    /// @param symbol       Symbol of the token
+    /// @param token        Address of the token
+    /// @param relayerFee   Fee % for bridging a token to this chain, multiplied by `FEE_DENOMINATOR`
+    /// @param minBaseFee   Minimum fee for bridging a token to this chain using a base request
+    /// @param minSwapFee   Minimum fee for bridging a token to this chain using a swap request
+    /// @param maxFee       Maximum fee for bridging a token to this chain
+    function addToken(
+        string memory symbol,
+        address token,
+        uint256 relayerFee,
+        uint256 minBaseFee,
+        uint256 minSwapFee,
+        uint256 maxFee
+    ) external onlyOwner {
+        if (token == address(0)) revert CCTPIncorrectConfig();
+        // Add a new token to the list of supported tokens, and check that it hasn't been added before
+        if (!_bridgeTokens.add(token)) revert CCTPTokenAlreadyAdded();
+        // Check that symbol hasn't been added yet and starts with "CCTP."
+        _assertCanAddSymbol(symbol);
+        // Add token <> symbol link
+        tokenToSymbol[token] = symbol;
```

Comment:
Any reason for storing `symbol`? Can't seem to find where this is used elsewhere in the contracts

Reply:
`tokenToSymbol[token]` is used in L175 for `getBridgeTokens()`, which the CCTP Bridge Module could later use for integrating. 

We are/will be using symbols to represent a bridge token deployed on a variety of chains. So for instance, `USDC` symbol will represent USDC locked in Synapse Bridge contract, while `CCTP.USDC` will represent USDC bridged using CCTP module. The set of available chains will be different for these symbols, this is why we use symbols rather than token addresses. And we will use different symbols for SynBridge / SynCCTP as well.